boot/bootutil/include/bootutil/sha256.h

```diff
@@ -31,11 +31,11 @@
 
 #include "mcuboot_config/mcuboot_config.h"
 
-#if defined(MCUBOOT_USE_MBED_TLS) && defined(MCUBOOT_USE_TINYCRYPT)
+#if defined(MCUBOOT_USE_MBED_TLS) && defined(MCUBOOT_USE_TINYCRYPT) && defined(MCUBOOT_USE_CC310)
```

Comment:
Will this work if I define `MCUBOOT_USE_MBED_TLS` and `MCUBOOT_USE_TINYCRYPT` but not `MCUBOOT_USE_CC310`? (I don't think this is totally right!)

Reply:
Also you need to mention `MCUBOOT_USE_CC310`

Reply:
Ah, yeah I see I need to do some clauses around. It will get pretty complex in the end

Reply:
Fixed ( I think)